src/url.rs

```diff
@@ -5,7 +5,7 @@ use std::error::Error;
 use url::{form_urlencoded, Url as UrlI};
 
 #[derive(Clone, Debug)]
-pub struct Url(pub String);
+pub struct Url;
```

Comment:
This string field is dead code, Clippy was complaining about it so I removed it to avoid breaking the build.